help/print-a-topic.md

```diff
@@ -0,0 +1,12 @@
+# Print a topic
+
+Topics are printable in Zulip. To save paper, printed topics exclude sidebars
+and other UI features that you otherwise see on screen. And whether you're
+viewing Zulip in the [light or dark theme](/help/dark-theme), Zulip topics will
+print with appropriate colors, or their grayscale equivalents, for paper or
+system-generated PDFs. Background colors will not be printed.
```

Comment:
I don't know if we need the information about the colors, grayscale, background here. Presumably this would be evident when the user sees the print preview.

Maybe a simpler note about how a user's setting for the light or dark theme will not impact the print colors or formatting?

Reply:
Maybe? I think the description helps document the feature a bit--and also relieves any concerned users who fear they'll burn through their ink or toner. I've moved the copy below the new instruction blocks. Maybe it's okay that way?